app/controllers/annotation_controller.js

```diff
@@ -0,0 +1,168 @@
+import Annotation from '../models/annotation';
+
+export const createAnnotation = (req, res) => {
+  if (req.isAuthenticated()) {
```

Comment:
Hmm this is a good question though -- because I think @psloomis is right in that this would be really annoying to have this everywhere...